Bluepill-cli/Bluepill-cli/Bluepill/Bluepill.m

```diff
@@ -159,11 +193,12 @@ - (void)proceed {
         return;
     }
     self.retries += 1;
-    [BPUtils printInfo:INFO withString:@"Exit Status: %@", [BPExitStatusHelper stringFromExitStatus:self.finalExitStatus]];
+    [BPUtils printInfo:INFO withString:@"Exit Status: %@", [BPExitStatusHelper stringFromExitStatus:self.context.exitStatus]];
     [BPUtils printInfo:INFO withString:@"Failure Tolerance: %lu", self.failureTolerance];
     [BPUtils printInfo:INFO withString:@"Retry count: %lu", self.retries];
     self.context.attemptNumber = self.retries + 1; // set the attempt number
     self.context.exitStatus = BPExitStatusTestsAllPassed; // reset exitStatus
+    [BPUtils printInfo:INFO withString:@"Start to proceed"];
```

Comment:
It is not a leftover message. I prefer to keeping this, because sometimes we don't know which branch it goes